docs/install-config/configure-internal-tls.md

```diff
@@ -9,7 +9,7 @@ This functionality is introduced via the `internal_tls` in `harbor.yml` file. To
 
 All certs can be automatically generated by `prepare` tool.
 ```bash
-docker run -v /:/hostfs goharbor/prepare:v2.0 gencert -p /path/to/internal/tls/cert
+docker run -v /:/hostfs goharbor/prepare:v2.1 gencert -p /path/to/internal/tls/cert
```

Comment:
Should this be `goharbor/prepare:v2.1.0` ?

Reply:
yup!